locales/en/translation.json

```diff
@@ -464,19 +569,24 @@
   "Tidepool Applications Terms of Use": "YourLoops Applications Terms of Use",
   "Tidepool is stuck and isn't doing what you want it to do. We're sorry for the trouble.": "YourLoops is stuck and isn't doing what you want it to do. We're sorry for the trouble.",
   "Tidepool provides free, secure data storage for diabetes data.": "YourLoops provides free, secure data storage for people using a Diabeloop system.",
-  "Tidepool Uploader": "",
+  "Tidepool Uploader": "Uploader",
   "Tidepool will attempt to send the details to our server.": "YourLoops will attempt to send the details to our server.",
-  "Tidepool will share 10% of the proceeds with the diabetes organization(s) of your choice.": "YourLoops:",
+  "Tidepool will share 10% of the proceeds with the diabetes organization(s) of your choice.": "YourLoops",
   "Tidepool's visualizations are only certified to work in the Chrome browser, and on Mac or PC.": "YourLoops's visualizations are only certified to work in the Chrome browser, and on Mac or PC.",
-  "Time In {{automatedLabel}}": "",
-  "Time in {{automatedLabel}} ratio": "",
+  "Time Above Range": "",
+  "Time Below Range": "",
+  "Time in {{automatedLabel}}": "Time in loop mode",
```

Comment:
The `{{automatedLabel}}` parameter should be kept in the translation.

Reply:
It's a choice to remove it in the translations so that we use only the "loop mode" wording.